client/src/EMS/RingdownForm.js

```diff
@@ -95,13 +95,25 @@ function RingdownForm({ className }) {
     setRingdowns([...ringdowns]);
   }
 
+  function onSubmit(event) {
+    event.preventDefault();
```

Comment:
@fwextensions what you had implemented didn't fix the original bug for me- pressing enter still resulted in the browser doing a standard form POST (resulting in reloading the webapp back to an empty form).

So, I changed it so that we ALWAYS preventDefault on the form submit action, and that fixes it.

Although, now I'm wondering if we should just put this into the Form component itself and have it do that on every submit (since I doubt we'll ever want standard browser form behavior)?

What do you think?

Reply:
I'm going to make the change to move the event.preventDefault into the Form component, then merge this in...